evm/src/extmock.rs

```diff
@@ -90,8 +84,9 @@ impl ext::DataProvider for DataProviderMock {
             .insert(key, value);
     }
 
-    fn selfdestruct(&mut self, address: &Address, _: &Address) {
+    fn selfdestruct(&mut self, address: &Address, _: &Address) -> bool {
         self.db.remove(address);
+        true
```

Comment:
If it always returns true, why do you keep it?

Reply:
`extmock` is used for unittest. In fact, it can be false, See: https://github.com/mohanson/cita-vm/blob/executive/src/executive.rs#L569